src/features/DebugSession.ts

```diff
@@ -80,6 +80,12 @@ export class DebugSessionFeature implements IFeature {
                     }
                 }
             }
+            else if (config.script) {
+                // In this case, the user has explicitly defined a script path
+                // so make sure to set the cwd to that path if the cwd wasn't
+                // explicitly set
+                config.cwd = config.cwd || config.script;
```

Comment:
OK so if we pass undefined or null we get to this code:
```csharp
            string workingDir =
                launchParams.Cwd ??
                launchParams.Script ??
#pragma warning disable 618
                launchParams.Program;
#pragma warning restore 618
```
That tries to set the CWD to untitled:Untitled-1.  Then we try to get file attrs on that and it throws. In the catch handler, we reset workingDir to `null` and then later that gets set to `Environment.CurrentDirectory` which is initially $Home.  But that can change in the integrated console as the user cd's around.  Not sure if that is a feature or a problem.  Anyway, specifically for unsaved files, seems like we should handle setting the working dir on the PSES side since it eventually knows where it will save the text in order to run it.  I assume you actually save the unsaved file contents to a file to run it.  Is that correct?  If so, then we could set the CWD to path holding the temp file. Do you think that makes sense? I'm not so sure. It is unlikely that would be advantageous.  So maybe we force it $home (always writable and any relative path file output generated by the script would be easier to find) or we stick with Environment.CurrentDirectory.  

If we go with home then we should be able to specify that on the extension side ${env.HOME} or something like that (assuming we find a xplat env var).  OK that's a lot of yammering.  Your thoughts?